packages/@aws-cdk/aws-stepfunctions-tasks/lib/base-run-task.ts

```diff
@@ -0,0 +1,298 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import iam = require('@aws-cdk/aws-iam');
+import stepfunctions = require('@aws-cdk/aws-stepfunctions');
+import cdk = require('@aws-cdk/cdk');
+
+/**
+ * Properties for SendMessageTask
+ */
+export interface CommonRunTaskProps extends stepfunctions.BasicTaskProps {
```

Comment:
I kind of expected this to be under some `ecs.ts` and have some ECS context in the interface name